askama_derive/Cargo.toml

```diff
@@ -37,4 +37,4 @@ proc-macro2 = "1"
 quote = "1"
 serde = { version = "1.0", optional = true, features = ["derive"] }
 syn = "1"
-toml = { version = "0.5", optional = true }
+toml = { version = "0.6", optional = true }
```

Comment:
So this should go in the next commit, right?

Reply:
This change was not need at all anymore, because the next commit replaces `toml` with `toml_edit`. I removed the change.

Reply:
The point of the first commit was to change the code to not depend on types borrowing data from the input TOML `&str` without changing any other code (for the update).